wiki-jpa-dao/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml

```diff
@@ -51,11 +52,11 @@
         <constraints nullable="false"/>
       </column>
       <column name="OWNER" type="NVARCHAR(200)"/>
-      <column name="CREATED_DATE" type="TIMESTAMP" defaultValueDate="${now}">
-        <constraints nullable="false"/>
+      <column name="CREATED_DATE" type="TIMESTAMP">
```

Comment:
We can keep one but in the later changeSet we will replace all of them to NOT NULL and default is '1970-01-01 00:00:00'
So, I set it to nullable for easier in reading and keep consistence between rdbms by avoid define ${now}